lib/fs_utils/asset.js

```diff
@@ -66,4 +74,56 @@ class Asset {
   }
 }
 
+class CompiledAsset {
+  constructor(path, publicPath, assetsConvention, compiler) {
+    if (compiler.length !== 1) {
+      throw new Error(`Got ${compiler.length} compilers for static file ${path}, can't proceed`);
```

Comment:
Since assets are only meant to be processed with one compiler, this will blow up if there ever ends up being more that one... the alternative could be to use the first compiler, and print a warning. @paulmillr what do you think?